INSTALL.md

```diff
@@ -122,10 +122,10 @@ devkitARM is now installed.
 
 To install Python on WSL1, simply run the following commands:
 
-    ```bash
-    sudo apt update && upgrade
-    sudo apt install python3
-    ```
+```bash
+sudo apt update && sudo apt upgrade
+sudo apt install python3
+```
```

Comment:
how about
```suggestion
```bash
sudo apt update
sudo apt upgrade
sudo apt install python3
```
```

Reply:
I'm ambivalent personally, but I would note we already use the &&-style version of those commands in the WSL install instructions. So it's probably better if we change both or neither for consistency, IMO.

Reply:
yeah, it doesn't really matter.

Reply:
There is no benefit in using the commands separately as opposed to having the user run them one after the other by virtue of the ampersand.

I'd also like to say that there shouldn't be a need for the user to run `sudo apt update && sudo apt upgrade` only before installing Python, or specifically in order *to* install Python.

`sudo apt update && sudo apt upgrade` is something the user should realistically only run once after setting up WSL and a valid Linux distro, and then once in a while *(and that isn't even mandatory.)*